examples/plot_partial_dependence_visualization_api.py

```diff
@@ -0,0 +1,123 @@
+"""
+=========================================
+Advanced Plotting With Partial Dependence
+=========================================
+The :func:`~sklearn.inspection.plot_partial_dependence` function returns a
+:class:`~sklearn.inspection.PartialDependenceDisplay` object that can be used
+for plotting without needing to recalculate the partial dependence. In this
+example, we show how to plot partial dependence plots and how to quickly
+customize the plot with the Visualization API.
```

Comment:
Do we use Display API now? (also example file name?)

Reply:
I have not changed the name of how to reference the API. I would still say "visualization API". "Display API" sounds overly generic.

Reply:
Ok, then lowercase (I thought this was a class ^^)